kafkaesk/app.py

```diff
@@ -46,10 +48,17 @@
 
 
 class Subscription:
-    def __init__(self, stream_id: str, func: Callable, group: str):
+    def __init__(
+        self,
+        stream_id: str,
+        func: Callable,
+        group: str,
+        retry_handlers: Optional[Dict[Type[Exception], RetryHandler]] = None,
```

Comment:
Can we make this required to add named param?
```suggestion
        *,
        retry_handlers: Optional[Dict[Type[Exception], RetryHandler]] = None,
```

Reply:
And done.